### PR TITLE
Feature: add onboarding screens

### LINKED ISSUE #7: Feature: onboarding screens
### What do with this issue

 Add onboarding screens for the medical app to ensure compatibility with both dark and light modes. 

### Description of Task to be completed?

The task involved creating onboarding screens and making necessary adjustments to ensure they were visually appealing and functional in both dark and light modes. This includes modifying color schemes, updating text styles, and adjusting layout spacing where necessary.

### PR DESCRIPTION
**Pull request by Irirwanirira**
#### What does this PR do?
This pull request adds updates to the onboarding screens of the medical app to ensure compatibility with both dark and light modes. It includes adjustments to the UI elements, colors, and layouts to maintain consistency and readability across different modes.

#### Description of Task to be completed?

The task involved reviewing the existing onboarding screens and making necessary adjustments to ensure they are visually appealing and functional in both dark and light modes. This includes modifying color schemes, updating text styles, and adjusting layout spacing where necessary.

#### How should this be manually tested?

These changes could be tested by running the app on your Android Studio emulator or by having the Expo app installed on your phone and scanning the QR code to try the app.

1. Open the app in light mode and go through the onboarding process.
2. Verify that all text and UI elements are visible and readable.
3. Switch to dark mode and repeat the onboarding process.
4. Ensure that the onboarding screens maintain their usability and readability in dark mode as well.
5. Test on different devices with varying screen sizes and resolutions to ensure consistency.

#### Any background context you want to provide?
The update to the onboarding screens was necessary to enhance the user experience and ensure accessibility for users who prefer dark mode. By making these adjustments, we aim to provide a seamless onboarding experience for all users, regardless of their preferred mode.

#### Screenshots

<div style="display: flex; flex-wrap: wrap;">
    WhatsApp Image 1
    WhatsApp Image 2
</div>

### CODE REVIEW

**Review by mr3nz1 — APPROVED** [review]

**Review by Franklin-pro — APPROVED** [review]

File: app.json [hunk 1]
```diff
@@ -2,10 +2,10 @@
   "expo": {
     "name": "matadors-rn-medica",
     "slug": "matadors-rn-medica",
-    "scheme": "matadors-rn-medica",
     "version": "1.0.0",
     "orientation": "portrait",
     "icon": "./assets/images/icon.png",
+    "scheme": "myapp",
```
Comment (Nkbtemmy) [inline app.json]: please let have common schema

File: app/onBoarding/_layout.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  const colorScheme = useColorScheme();
+
+  return (
+    <Stack
+      // screenOptions={{
+      //   tabBarActiveTintColor: Colors[colorScheme ?? "light"].tint,
+      //   // Disable the static render of the header on web
+      //   // to prevent a hydration error in React Navigation v6.
+      //   headerShown: useClientOnlyValue(false, true),
+      // }}
```
Comment (Nkbtemmy) [inline app/onBoarding/_layout.tsx]: In case you don't need codes; remove them in pushed codes

**Review by Nkbtemmy — CHANGES_REQUESTED** [review]
🔕 please squash your commits: https://www.youtube.com/watch?v=viY1BbKZhSI

**Comment by Nkbtemmy** [comment]
At the end; please change this PR label to need the Review label

**Review by mr3nz1 — APPROVED** [review]

**Review by Nkbtemmy — APPROVED** [review]

**Pull request merged**